test/includes/lxc.sh

```diff
@@ -0,0 +1,52 @@
+# lxc CLI related test helpers.
```

Comment:
So either this commit includes a lot more than it's meant to, or it's poorly named because the changes below include a lot more than just the network helper functions.